teachertool/src/state/state.ts

```diff
@@ -0,0 +1,12 @@
+import { Notifications } from "../types";
+
+export type AppState = {
+    targetConfig?: pxt.TargetConfig;
+    notifications: Notifications;
+    volume?: number; // volume level of UI sounds, in [0..1]
```

Comment:
I imagine we'll remove this `volume` setting.

Reply:
I just went ahead and removed it now